### DISCUSSION TITLE
Choice of power pack -- does splitter change power supplied to arduino and shield?

### DISCUSSION THREAD

**Discussion by fiscalia8** [pdbuzzboard/pdbuzzboard#18] [Q&A] [answered]
Hi folks, 
I'm preparing to make a version of this for my dad. I have some experience with mechatronics, so this project looks like a lot of fun, and I'm hoping to improve cable management along the way... but electrical current is my weakest engineering subject, and I am worried about blowing the motors. 
My question is whether the USB power splitter provides functionally less power to each of the two devices?
I'm considering a rechargable USB power pack [like this one](https://www.amazon.com/dp/B0D1JYZF69) that has two USB output. Would omitting the splitter and plugging the arduino and shield directly into individual ports mess me up?
Thanks!

**Comment by pdbuzzboard** [comment, marked as answer]
The splitter should not reduce the power appreciably.  That being said, I had one batch of "cheap Chinese" splitters that did drop power significantly, but they were defective product.  The primary drop point from 5V to 3.xV is in the motorshield due to the "efficiency" of the chipset on the device.
I chose the splitter approach in order to have a single switch for the entire device.  Most of the power supplies that I use have dual ports, but the on off switch on some of them can be a bit cumbersome.

**Reply by fiscalia8** [reply]
Got it. Thanks for the reply!

**Reply by garckenn** [reply]
I'm trying these two adapters to eliminate the need for a splitter. (i.e. wiring the Barrel plug adapter from the Arduino directly to the USB pigtails that are wired to the Motor Shield, in such that they are powered in parallel without using a USB splitter)

- https://www.amazon.com/dp/B0BLKMDM66
- https://www.amazon.com/dp/B07C61434H

![image](https://github.com/user-attachments/assets/20cd6b5d-6ee7-4e0e-a4b0-b474932743fb)


![51TO5S+4LFL _SL1500_](https://github.com/user-attachments/assets/acbd2e32-0e6a-4887-8d1e-df9fffc4a8ae)

**Comment by falkonline** [comment]
I too initailly rigged up a battery system like this, but ended up get plugging the boards to all my spare USB to wall power adaptors (in a powerbar with a long cable). My dad doesn't move  that well anyway, and struggles to turn off anything electric. The added complications of batteries (charging, carring around, charging, more cable tangles, etc) wasnt worth it.
